fedgov-cv-message-validator-webapp/src/main/java/gov/usdot/cv/service/rest/Configuration.java

```diff
@@ -0,0 +1,55 @@
+package gov.usdot.cv.service.rest;

```

Comment:
please add this copyright statement to all new .java files:

/*
 * Copyright (C) 2025 LEIDOS.
 *
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not
 * use this file except in compliance with the License. You may obtain a copy of
 * the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
 * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 * License for the specific language governing permissions and limitations under
 * the License.
 */